GamepadHapticsActuatorTriggerRumble/explainer.md

```diff
@@ -96,9 +97,11 @@ Allowing websites to query for Gamepads' haptic capabilities might raise fingerp
 
 ## Alternative Solutions
 
-### `canPlay` versus a new `GamepadHapticActuatorType` entry
+### `GamepadHapticActuatorType` entry for effect support detection
+Adding a new entry entry, e.g. "trigger-rumble" to GamepadHapticActuatorType might be difficult to accomplish without breaking existing apps. For example, an app that always checks for "dual-rumble" may break if Xbox controllers suddenly begin to return "trigger rumble" as their `GamepadHapticActuatorType`. Therefore, it may be better to let the web applications do the required capability detection using a different route - e.g., by calling a `canPlay` method -; and always define `GamepadHapticActuator.type` to be "dual-rumble", so apps cannot rely on it, then deprecate, and eventually remove it.
 
-Adding a new entry entry, e.g. "trigger-rumble" to GamepadHapticActuatorType might be difficult to accomplish without breaking existing apps. For example, an app that always checks for "dual-rumble" may break if Xbox controllers suddenly begin to return "trigger rumble" as their `GamepadHapticActuatorType`. Therefore, it may be better to let the web applications do the required capability detection by calling `canPlay`; and always define `GamepadHapticActuator.type` to be "dual-rumble", so apps cannot rely on it, then deprecate, and eventually remove it.
+### `canPlay` method
+A new `canPlay` method that takes a `GamepadHapticEffectType` string and returns a `boolean` could also be used to query a gamepad's haptics capabilities. However, this requires a new call to be made for every `GamepadHapticEffectType` we would like to check for support. Moreover, implementing a method like this could affect legacy engines, case effect types need to be deprecated or renamed.
```

Comment:
nit: I'm not sure I understand the following:

"Moreover, implementing a method like this could affect legacy engines, case effect types need to be deprecated or renamed."

Also, I think maybe there's a typo in there where "case" should be "in case".

Reply:
For example, if at some point we decide to deprecate `dual-rumble`, websites that used to call `canPlay('dual-rumble')` would start getting an exception. With the new array approach, we wouldn't have this problem.

Reply:
More context on this here: https://github.com/w3c/gamepad/pull/163#pullrequestreview-1000657433

Reply:
ok, thanks for clarifying.  I didn't expect unknown enum values for canPlay() to throw an exception, but I guess that makes sense.